config/buildout-recipe/tox.ini.j2

```diff
@@ -36,7 +36,7 @@ commands =
 {% endif %}
     coverage combine
     coverage html
-    coverage report -m --fail-under=%(coverage_fail_under)s
+    coverage report
```

Comment:
The command-line switches shadow and/or overwrite configuration values. That makes no sense because now you have to be aware of configuration in two places, the configuration file itself and the command invocation. With this PR, the configuration file becomes the only place where configuration is stored. The invocation works correctly without these switches.

Reply:
👍